blog/2015/10/python-debriefing-summary.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+root: ../../..
+author: Alistair Walsh
+title: "A Summary of Debriefing Feedback on Our Python Lesson"
+date: 2015-10-08
+time: "01:00:00"
+category: ["Lessons", "Python"]
+---
+<!-- start excerpt -->
+<p>
+  Last month,
+  we discussed results from a survey of
+  <a href="{{page.root}}/blog/2015/09/thinking-about-teaching.html">how our instructors are teaching Python</a>.
+  We now have a summary of the feedback we've received in our bi-weekly debriefing meetings.
+  The recurring themes are:
+</p>
+<ol>
+  <li><p>A greater choice of exercises and multiple choice questions would allow instructors to select domain specific examples and cater to varying levels of learner.</p></li>
+  <li><p>Some instructors added an explanation of the IPython/Jupyter/SpyderIDE environment.</p></li>
```

Comment:
"IPython/Jupyter/SpyderIDE" -> "IPython/Jupyter or SpyderIDE" just because IPython and Jupyter are the name for the "same" environment but SpyderIDE is one completely different.